api.yml

```diff
@@ -185,66 +83,166 @@ paths:
-          type: object
-          additionalProperties: true
-          description: Optional object containing claims and their values.
-          example: {"name": "Jayden Doe"}
-        evidence:
+          description: A reference to the issuer's identifier.
+        credential:
           type: object
-          additionalProperties: true
-          description: Optional object containing evidence of the credential.
-          example: {}
-        termsOfUse:
-          type: object
-          additionalProperties: true
-          description: Optional object containing terms of use of the credential.
-          example: {}
-        credentialFormat:
+          description: The credential to be issued minus the proofs
+        output:
+          $ref: '#/components/schemas/Output'
+      example: {
+        "issuer": "did:example:1234567",
+        "credential": {
+          "credential": {
+            "@context": [
+              "https://www.w3.org/2018/credentials/v1",
+              "https://w3id.org/citizenship/v1"
+            ],
+            "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
+            "type": [
+              "VerifiableCredential",
+              "PermanentResidentCard"
+            ],
+            "issuanceDate": "2019-12-03T12:19:52Z",
+            "expirationDate": "2029-12-03T12:19:52Z",
+            "credentialSubject": {
+              "id": "did:example:b34ca6cd37bbf23",
+              "type": [
+                "PermanentResident",
+                "Person"
+              ],
+              "givenName": "JOHN",
+              "familyName": "SMITH",
+              "gender": "Male",
+              "image": "data:image/png;base64,iVBORw0KGgo...kJggg==",
+              "residentSince": "2015-01-01",
+              "lprCategory": "C09",
+              "lprNumber": "999-999-999",
+              "commuterClassification": "C1",
+              "birthCountry": "Bahamas",
+              "birthDate": "1958-07-17"
+            }
+          }
+        },
+        "output": {
+          "encoding": "jsonld",
+          "encodingOptions": {
+            "suite": "JsonWebSignature2020",
+            "alg": "ES256"
+          }
+        }
+      }
+    CredentialResponse:
+      type: object
+      properties:
+        credentialReference:
           type: string
-          description: Optional value to specify the credential format. Well-known values are "jsonld", "jwt", "blockcerts". Other values MAY be supported. If this is missing, it may be pre-configured, and a default credential format will be used.
-          example: jsonld
-        proofFormat:
+          description: Reference to the issued credential.
+        credential:
+          description: Encoded credential.
+          oneOf:
+            - type: string
+            - type: object
+    Claims:
+      type: object
+      description: A set of claims about a single subject. A claim without properties other than 'sub' MUST be rejected.
```

Comment:
This was what I was concerned about when I noted that there seems to be a propensity in some of the current proposals for inventing new data structures when equivalent ones exist in the Verifiable Credentials Data Model... for example, `id` exists for this purpose in the Verifiable Credentials Data Model. We should be using `id` here instead of `sub`.

Reply:
Can be changed.